hax/__init__.py

```diff
@@ -23,6 +24,13 @@
 # Just do import hax, then hax.init(), then access config has hax.config.
 config = {}
 
+# This stores the unblinding selection (do not try to modify directly; unblinding.ini
+# file should be modified with proper PR and review if needed)
+unblinding_config = {}
+
+# Flag for whether you are working with MC (True) or not
+mc_data = False
```

Comment:
Could you clarify what this global mc_data is doing? It looks like this is a run-level property, but it's stored as a package global. You might get some interesting behaviour if you're doing real data / MC comparison studies.  Could the treemakers that need to know whether they are in MC land (or maybe event the treemaker base class) call hax.runs.is_mc instead?

Reply:
Remnant from confusion during development. Fixed in 1154d04.